manifest-novo.json

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
This was added automatically – I'm assuming I should remove from the PR, but would like to know why it keeps being generated

Reply:
@damassi? First I've ever seen this.

Reply:
This is ancient! I think it might be from your old copy of force, before all this was cleared up (and therefore might be missing from gitignore? 

History is: we had two parallel webpack configs going temporarily while we figured out how to migrate away from old config for v2 work.

Reply:
😅 Okay cool, that tracks!! Thanks @damassi 🙏